mycore-base/src/main/java/org/mycore/datamodel/common/MCRFileBaseCacheObjectIDGenerator.java

```diff
@@ -0,0 +1,173 @@
+package org.mycore.datamodel.common;
```

Comment:
Please Provide MyCoRe License Comment + Javadoc for class

Reply:
added both